ui/app/src/Utils/Date.elm

```diff
@@ -62,14 +65,19 @@ durationFormat time =
         |> String.trim
 
 
-dateFormat : ISO8601.Time -> String
```

Comment:
Maybe smth for the future, I think it makes sense to only have one datetime library